app/gui2/src/components/GraphEditor.vue

```diff
@@ -246,10 +246,11 @@ const graphBindingsHandler = graphBindings.handler({
 
 const { handleClick } = useDoubleClick(
   (e: MouseEvent) => {
-    graphBindingsHandler(e)
+    if (e.target !== viewportNode.value) return false
```

Comment:
You can check `e.target !== e.currentTarget`, avoiding the requirement of vue ref.